go.mod

```diff
@@ -6,28 +6,28 @@ require (
 	github.com/ViaQ/logerr/v2 v2.1.0
 	github.com/go-logr/logr v1.4.2
 	github.com/open-telemetry/opentelemetry-operator v0.107.0
-	github.com/openshift/cluster-logging-operator/api/observability v0.0.0-20241106184417-73a5857dc558
+	github.com/openshift/cluster-logging-operator/api/observability v0.0.0-20250228165433-3665fb550c08
 	github.com/operator-framework/api v0.26.0
-	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.75.2
-	github.com/spf13/cobra v1.8.1
-	github.com/spf13/pflag v1.0.5
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.76.2
+	github.com/spf13/cobra v1.9.1
+	github.com/spf13/pflag v1.0.6
 	github.com/stolostron/cluster-lifecycle-api v0.0.0-20240918064238-a5e71b599118
-	github.com/stretchr/testify v1.9.0
+	github.com/stretchr/testify v1.10.0
 	k8s.io/api v0.30.3
 	k8s.io/apiextensions-apiserver v0.30.3
 	k8s.io/apimachinery v0.30.3
 	k8s.io/client-go v0.30.3
 	k8s.io/component-base v0.30.3
 	k8s.io/klog/v2 v2.130.1
-	k8s.io/utils v0.0.0-20240502163921-fe8a2dddb1d0
+	k8s.io/utils v0.0.0-20240711033017-18e509b52bc8
 	open-cluster-management.io/addon-framework v0.11.0
 	open-cluster-management.io/api v0.15.0
 	sigs.k8s.io/controller-runtime v0.18.5
 )
 
-replace github.com/prometheus/common => github.com/prometheus/common v0.60.1
+replace github.com/prometheus/common => github.com/prometheus/common v0.62.0
 
-replace github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.20.5
+replace github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.21.0
```

Comment:
It doesn't look like you need the `replace` lines. I think I'd remove them and update the versions of the `indirect` dependencies below.